developer/developer-faqs.md

```diff
@@ -1,6 +1,14 @@
-# Frequently asked questions
-
+# Frequently Asked Questions
 ## General
+### What do I need to know to work with Reaction?
+
+To start hacking Reaction, you should probably have a basic understanding of the following technologies:
```

Comment:
suggest "customizing" vs "hacking"

Reply:
changed. but  hacking does not carry bad intentions, does it?  (at least not among devlopers)